python_modules/libraries/dagster-dbt/dagster_dbt/asset_selection.py

```diff
@@ -42,6 +42,10 @@ class DbtManifestAssetSelection(AssetSelection):
                 node_info_to_asset_key=my_node_info_to_asset_key_fn,
             )
 
+            # This will retrieve the asset keys according to the selection
+            asset_graph = AssetGraph.from_assets(dbt_assets)
+            selected_asset_keys = my_selection.resolve_inner(asset_graph)
```

Comment:
this should be `my_selection.resolve(asset_graph)`, not `resolve_inner`. The former is the public interface, the latter is the private one that all AssetSelection objects override.

`resolve` also has the benefit of accepting `Union[Sequence[Union[AssetsDefinition, SourceAsset]], AssetGraph]`, rather than just `AssetGraph`, so this could just be `my_selection.resolve(dbt_assets)`